sdk/identity/identity/samples/AzureIdentityExamples.md

```diff
@@ -908,6 +915,153 @@ class RotatingCertificateCredential implements TokenCredential {
 
 In this example, the custom credential type `RotatingCertificateCredential` again uses a `ClientCertificateCredential` instance to retrieve tokens. However, in this case, it will attempt to refresh the certificate before obtaining the token. The method `RefreshCertificate` will query to see if the certificate has changed. If so, it will replace `this.credential` with a new instance of the certificate credential using the same certificate path.
 
+### Controlling User Interaction
+
+In many cases applications require tight control over user interaction. In these applications automatically blocking on required user interaction is often undesired or impractical. For this reason, credentials in the @azure/identity library which interact with the user offer mechanisms to fully control user interaction.
+
+```
+  const options: InteractiveBrowserCredentialOptions = {
+    disableAutomaticAuthentication: true
+  };
+  const credential = new InteractiveBrowserCredential(options);
+  await credential.authenticate("https://vault.azure.net/.default");
+  const client = new SecretClient("https://key-vault-name.vault.azure.net", credential);
+```
+
+In this sample the application is again using the `InteractiveBrowserCredential` to authenticate a SecretClient, but with two major differences from our first example. First, in this example the application is explicitly forcing any user interaction to happen before the credential is given to the client by calling `authenticate`.
+
```

Comment:
I don’t remember if this method is available both in Node and in the browser! Please help me verify. If this isn’t available in the browser, we should say that :)

Reply:
Yes it is available in both node and browser